x-pack/plugins/stack_connectors/server/connector_types/pagerduty/index.ts

```diff
@@ -79,6 +79,9 @@ const PayloadSeveritySchema = schema.oneOf([
   schema.literal('info'),
 ]);
 
+const LinksSchema = schema.arrayOf(schema.object({ href: schema.uri(), text: schema.string() }));
```

Comment:
I'm not sure we use `schema.url()` much - I'm thinking that it may have caused problems for us in the past, and we just used `schema.string()` instead.

Reply:
Done in https://github.com/elastic/kibana/pull/170459/commits/a1f5a564c708877d8be1a00cfbc40c5170cf0eb0